gossip/service.go

```diff
@@ -269,7 +270,7 @@ func newService(config Config, store *Store, blockProc BlockProc, engine lachesi
 	}
 
 	// create API backend
-	svc.EthAPI = &EthAPIBackend{config.ExtRPCEnabled, svc, stateReader, txSigner, config.AllowUnprotectedTxs}
+	svc.EthAPI = &EthAPIBackend{false, svc, stateReader, txSigner, config.AllowUnprotectedTxs}
```

Comment:
it seems that we intend to init this with `true`? otherwise, it still work like before

```suggestion
	svc.EthAPI = &EthAPIBackend{true, svc, stateReader, txSigner, config.AllowUnprotectedTxs}
```

Reply:
It's initialized by SetExtRPCEnabled later